operations/mimir-mixin/config.libsonnet

```diff
@@ -232,7 +232,10 @@
             container_memory_rss{container=~"(%(ingester)s|%(mimir_write)s|%(mimir_backend)s)"}
               /
             ( container_spec_memory_limit_bytes{container=~"(%(ingester)s|%(mimir_write)s|%(mimir_backend)s)"} > 0 )
-          ) > %(allocationpercent)s
+          )
+          # Make sure that the pods are really all Mimir pods.
+          and on(namespace, pod, container) group(cortex_build_info{container=~"(%(ingester)s|%(mimir_write)s|%(mimir_backend)s)"}) by (namespace, pod, container)
```

Comment:
You shouldn't hardcode `namespace, pod, container` because these are configurable. In alert alerts we do something like this. Can we do the same here as well?

```
# Match only Mimir namespaces.
* on(%(aggregation_labels)s) group_left max by(%(aggregation_labels)s) (cortex_build_info)
```